README.md

```diff
@@ -61,7 +61,7 @@ You can also include it in a `requirements.yml` file and install it via `ansible
 ---
 collections:
   - name: community.digitalocean
-    version: 1.0.0
+    version: 1.1.1
```

Comment:
Because it's so easy to forget this one, I would simply remove the version here ;-)

Reply:
Yup, agreed remove this.

Reply:
Argh, forgot to stage `galaxy.yml`, and yes, I'll remove it from the `README.md`, good idea 😄